### PR TITLE
add duplicate placeholders for symbols without night version

### PR DESCRIPTION
**Pull request by Jonifmi**
Add dublicate symbol of symbols without night version, so there is a corresponding symbol for every smartsymbol (INT).

### CODE REVIEW

**Review by Kirtapa — APPROVED** [review]

**Comment by NALStudio** [comment]
I'm not an FMI contributor by any means so take my words with a grain of salt, but I'm not a huge fan of duplicating these symbols. When these symbols are missing from the repo, it actually signals to other people that these symbols don't exist and they should use the day variant instead. By adding these duplicates, you are essentially conveying to the user that this special night symbol exists (and more crucially, differs from the day version).

If multiple identical symbols are added, it makes it difficult for the users of this repo to decipher which symbols are duplicates and can be omitted (from a client application for example) to save space. I'd rather that there'd be better documentation on these missing symbols (basically a note saying that if missing, use the day version) but they would be left missing. The user can then decide themselves to either duplicate these or save storage space and handle the missing assets through code.

TL;DR: This PR would make the distinction between unique and duplicate symbols ambiguous.

**Comment by sirviot** [comment]
Thank you for the insight @NALStudio.

It is true that the duplicate symbols can take more space and make the download bigger. Adding them also enables users to download the assets and use them as-is without the need for much programming skills. 

At the time the icons are copies of one another, but this update also makes it possible to adjust them to be more suitable for visualizing night time weather phenomena.
